src/main/kotlin/com/devfloor/untitled/articleviewcount/domain/ArticleViewCountRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.devfloor.untitled.articleviewcount.domain
+
+import com.devfloor.untitled.article.domain.Article
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+/**
+ * 게시글 좋아요를 관리하는 repository
+ */
+@Transactional(readOnly = true)
+interface ArticleViewCountRepository : JpaRepository<ArticleViewCount, Long> {
+    fun findByArticle(article: Article): Optional<ArticleViewCount>
+
+    fun deleteByArticle(article: Article)
```

Comment:
다른 부분은 다 괜찮은 것 같고 이 부분 bulk연산 제거 관련 리뷰 반영하시면서 Transactional까지 제거된 것 같아요 !!
여기만 반영하시면 될거같습니다 !
```suggestion
    @Transactional
    fun deleteByArticle(article: Article)
```